common/bindfstools.py

```diff
@@ -0,0 +1,157 @@
+#    Copyright (C) 2016 Taylor Raack
+#
+#    This program is free software; you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation; either version 2 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License along
+#    with this program; if not, write to the Free Software Foundation, Inc.,
+#    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+import os
+import grp
+import subprocess
+import gettext
+
+import config
+import mount
+import logger
+import tools
+from exceptions import MountException
+
+_=gettext.gettext
+
+# TODO - quite a few shared methods between the various mount.MountControl subclasses. Perhaps move some common
+# methods into the base class?
```

Comment:
Sure. I did this already yesterday for the [pre|post]_u?mount_checks in commit 9e484938224a68a3e30fbef18270af960d454497. But this would also work for `_umount()` and `_check_fuse()`